ahnlich/server/src/main.rs

```diff
@@ -5,10 +5,85 @@ mod engine;
 mod errors;
 mod network;
 mod storage;
+use std::error::Error;
 
-#[cfg(test)]
-mod tests;
+use clap::{ArgAction, Args, Parser, Subcommand};
+use tokio::io::{AsyncBufReadExt, AsyncWriteExt, BufReader};
+use tokio::net::{TcpListener, TcpStream};
+
+async fn run_server() -> std::io::Result<()> {
+    let listener = TcpListener::bind("127.0.0.1:1369").await?;
```

Comment:
Make use of the port and host here

Reply:
That was the plan before you marked it as ready for review 👀